frontend/packages/container-security/src/models.ts

```diff
@@ -4,7 +4,7 @@ export const ImageManifestVulnModel: K8sKind = {
   kind: 'ImageManifestVuln',
   label: 'ImageManifestVuln',
   labelPlural: 'ImageManifestVuln',
-  apiGroup: 'secscan.quay.redhat.com',
+  apiGroup: 'operators.coreos.com',
```

Comment:
This is not the correct API group for this CRD.

Reply:
> This resource is intended to be read-only from the user perspective and only created/updated/deleted by the Operator.
> 
> To fix the 404, we would need to add a `Page/Route` to the `plugin.ts` that handles the `~/new` path so it doesn't assume that is the `:name` parameter.

Alright! Will do, thanks

Reply:
@spadgett @alecmerdler Added the  `Page/Route`  to handle  `~/new`, however the present of the path below makes it to throw 404. 

`https://github.com/openshift/console/blob/e4e61c9bf4764db9d3ab6c0da375a5b2402adc22/frontend/packages/container-security/src/plugin.ts#L87`

The path above looks similar to `https://github.com/openshift/console/blob/e4e61c9bf4764db9d3ab6c0da375a5b2402adc22/frontend/packages/container-security/src/plugin.ts#L100`. 
Wondering if both are required.